lms/static/sass/login.scss

```diff
@@ -97,11 +97,15 @@
   text-align: left;
   font-weight: 600;
   color: var(--color-008080);
-  text-decoration: underline;
+  border-bottom: 1px solid var(--color-008080);
+}
+
+.login__section a:hover {
+  text-decoration: none;
 }
 
 .login__section label {
-  color: var(--color-212529);
+  color: var(--color-374049);
```

Comment:
dziwi mnie taka zmienna, to w sumie po prostu utrudniona nazwa koloru, na przyszłość skoro Agata ma już przemyślane kolory mogłybyście z Paulą przemyśleć nazwy zmiennych i się nimi w sposób spójny posługiwać. Nie musi być przy okazji tego taska, ale przy następnych.

Reply:
> Do zmiany jest jeszcze ostylowanie podkreślenia adresu e-mail pomocy w oknie z "Nie pamietam hasła"

zrobione

Reply:
OK. Zmienimy przy kolejnym projekcie Agaty nazewnictwo kolorów